src/main/java/hudson/remoting/RemoteInvocationHandler.java

```diff
@@ -224,6 +226,7 @@ public static Channel unwrap(Object proxy) {
         return null;
     }
 
+    @Nullable
```

Comment:
This is the default, I assume you are adding it here to remind yourself not to go over-zealous and add `@CheckForNull` ;-)

Also I note that in this specific case, the assumption of `@Nullable` is entirely correct (if effectively redundant), we cannot know what the contract of the invoked method is w.r.t. nullability, rather the caller of the proxied interface should know the contract of the method they are requesting an invocation of.